docs/src/examples/components/Menu/Types/index.tsx

```diff
@@ -19,6 +19,16 @@ const Types = () => (
       description="A vertical menu displays elements vertically."
       examplePath="components/Menu/Types/MenuExampleVertical"
     />
+    <ComponentExample
+      title="Menu with submenus"
+      description="A menu can have submenus."
+      examplePath="components/Menu/Types/MenuExampleWithSubmenu"
+    />
+    <ComponentExample
+      title="Vertical menu with submenus"
+      description="A vertical menu can have submenu."
+      examplePath="components/Menu/Types/MenuExampleVerticalWithSubmenu"
```

Comment:
This example should be present, but we can leverage the attention to it there.

```diff
<ComponentExample
-  title="Vertical menu with submenus"
-  description="A vertical menu can have submenu."
  examplePath="components/Menu/Types/MenuExampleVerticalWithSubmenu"
/>
```

<details>
  <summary>Preview</summary>

![image](https://user-images.githubusercontent.com/14183168/50161705-c99bc380-02dc-11e9-848e-028263e23d70.png)
</details>